web/src/components/PoiPanelNavigation.tsx

```diff
@@ -48,7 +48,7 @@ const PoiPanelNavigation: React.FC<PoiPanelNavigationProps> = ({
 }: PoiPanelNavigationProps): ReactElement => {
   const { t } = useTranslation('pois')
   return (
-    <NavigationContainer>
+    <NavigationContainer data-testid='poi-panel-navigation'>
```

Comment:
I think you should rather add accessibility labels to the buttons and use those to test for this.

Reply:
you are right!

Reply:
`poi-panel-navigation` does not really make sense as aria label, does it? :see_no_evil: I think you should add something like `Previous` and `Next` as aria label to the buttons on not the container div.